compiler/dyno/lib/resolution/Resolver.cpp

```diff
@@ -792,6 +1239,30 @@ void Resolver::exit(const Dot* dot) {
   // TODO: resolve field accessors / parenless methods
 }
 
+/*
```

Comment:
I think you said you wanted to remove this, but honestly I'm kind of hoping that we could pull out the big blob of code that prepares the actuals and call it here. The reason being it just seems easier to manage.

Reply:
I'm open to it, but for now I will just remove this comment. We can better factor it in the future.